identity-verification/module_2/2.1 Rekognition IDV API Examples.ipynb

```diff
@@ -0,0 +1,706 @@
+    "-----\n",
+    "\n",
+    "There are several other APIs which we can take advantage of to manage and maintain our collection. \n",
+    "\n",
+    "<div class=\"alert alert-info\"><strong> Add'l APIs </strong>\n",
+    "\n",
+    "- ListCollections: Returns list of collection IDs in your account.\n",
+    "- DeleteFaces: Deletes faces from a collection. You specify a collection ID and an array of face IDs to remove from the collection\n",
+    "- ListFaces: Returns meta data for faces in the specified collection. This meta data includes information such as the bounding box coordinates, the confidence (that the bounding box contains a face), and face ID.\n",
+    "- DeleteCollection: Deletes the specified collection. Note that this operation removes all faces in the collection.\n",
+    "\n",
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### ListCollections \n",
+    "-----\n",
+    "ListCollections returns an array of Collection Ids and corresponding FaceModelVersions. This is helpful information to note when upgrading to newer versions of Rekognition's faces model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## -- list collections \n",
+    "response = rek_client.list_collections(\n",
+    "    MaxResults=123\n",
+    ")\n",
+    "\n",
+    "response"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### DeleteFaces \n",
+    "\n",
+    "------\n",
+    "Deletes faces from a collection. You specify a collection ID and an array of face IDs to remove from the collection. to do this snag two or more FaceIDs from the results above and simply pass them to the DeleteFaces API to remove them from the collection.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## -- Delete Faces\n",
+    "## Snag a few faceIDs from above. \n",
+    "response = rek_client.delete_faces(\n",
+    "    CollectionId=collection_name,\n",
+    "    FaceIds=[ \"\", \"\"]  # enter faceIDs here\n",
+    ")\n",
+    "response"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### What happens when you pass a faceID that doesn't exist? \n",
+    "----\n",
+    "Here we are simply going to search for a face id that we deleted, you'll note we get an error that the faceID does not exist"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Search for a face that doesn't exist \n",
+    "\n",
+    "face_id = \"48c04911-4573-439d-9389-bebc0994013a\"\n",
+    "try:\n",
+    "    response = rek_client.search_faces(\n",
+    "        CollectionId=collection_name,\n",
+    "        FaceId=face_id,\n",
+    "        MaxFaces=10\n",
+    "    )\n",
+    "except:\n",
```

Comment:
This should be scoped to Rekognition.Client.exceptions.ResourceNotFoundException